### PR TITLE
tick image to train track CA Respondent

### PR DESCRIPTION
**Pull request by BharadwajSairam**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/app/utils/progress-bar-utils.ts [hunk 1]
```diff
@@ -66,17 +66,26 @@ const buildProgressBarStages = (userCase: Partial<CaseWithId>): object => {
...
+    userCase.state === State.DECISION_OUTCOME ||
+    userCase.state === State.PREPARE_FOR_HEARING_CONDUCT_HEARING
+  ) {
+    return true;
+  }
+  return false;
+};
+
+const isResponseSubmitted = (userCase: Partial<CaseWithId>) => {
+  if (userCase.citizenResponseC7DocumentList) {
```
Comment (shashisk97) [inline src/main/app/utils/progress-bar-utils.ts:88]: How do we know the response is submitted by the one who logged in, if there are multiple respondents

File: src/main/steps/respondent/task-list/content.ts [hunk 2]
```diff
@@ -260,7 +260,6 @@ export const generateContent: TranslationFn = content => {
     content.userCase?.caseTypeOfApplication === 'C100'
       ? getC100Banners(content.userCase, translations, content.userIdamId)
       : getFl401Banners(content.userCase, translations, content.userIdamId);
-  console.log(JSON.stringify(buildProgressBarStages(content.userCase!)));
```
Comment (shashisk97) [inline src/main/steps/respondent/task-list/content.ts:263]: Please remove this

File: webpack/hmctsFrontend.js [hunk 3]
```diff
@@ -4,13 +4,19 @@ const CopyWebpackPlugin = require('copy-webpack-plugin');
...
-  patterns: [{ from: images, to: 'hmcts-assets/images' }],
+  patterns: [{ from: images, to: 'hmcts-assets/images' },
+  {
+    from: localImages,
+    to: 'assets/images',
+    noErrorOnMissing: true
+  }],
 });
-
+console.log(copyTemplateAssets);
```
Comment (shashisk97) [inline webpack/hmctsFrontend.js:19]: Please remove if its not needed

File: src/main/steps/respondent/task-list/content.ts (hunk 2 above)
Comment (BharadwajSairam) [reply src/main/steps/respondent/task-list/content.ts:263]: I removed it that's why it's showing that

**Review by swaroopa86 — APPROVED** [review]

**Pull request merged**